readthedocs/projects/forms.py

```diff
@@ -208,24 +208,23 @@ class ProjectAdvancedForm(ProjectTriggerBuildMixin, ProjectForm):
     class Meta:
         model = Project
         fields = (
-            # Standard build edits
-            'install_project',
-            'requirements_file',
-            'single_version',
-            'conf_py_file',
-            'default_branch',
+            # Global settings.
             'default_version',
+            'default_branch',
+            'privacy_level',
+            'analytics_code',
```

Comment:
This comment is outside the scope of this PR, but we could put this setting into the config file on a per-version basis. It might be nice to get that information versioned.

Reply:
Any per-version config, is excellent for the YAML. I agree with David to implement this in another PR.

Reply:
Don't people use just one analytics code per site? And also, isn't a way of identify each page from analytics?

Reply:
I opened https://github.com/rtfd/readthedocs.org/issues/5478